helloworld/index.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const greet = require('./lib/greet.js');
+const arithmetic = require('./lib/arithmetic.js');
+
+
+console.log(greet('bangarang'));
```

Comment:
Good job invoking your greet and arithmetic functions in your index.js file so that it was easier for me to see outputs. :)